nekoyume/Assets/StreamingAssets/Localization/item_name.csv

```diff
@@ -338,9 +338,5 @@ ITEM_NAME_40500009,Gray Tail,회색 꼬리,Cauda Cinzento,,灰色の尻尾,Graue
 ITEM_NAME_40500010,Blue Ghost Tail,파란 유령 꼬리,Cauda Azul Fantasmagórica,,青ゴーストの尻尾,Blauer Geisterschwanz,,หางผีสีฟ้า,Niebieski ogon ducha,,,Ekor Hantu Biru,Queue Fantômatique Bleue,,កន្ទុយខ្មោចខៀវ,,蓝鬼尾巴,
 ITEM_NAME_49900001,Legendary Adventurer,전설의 모험가,Aventureiro Lendário,,伝説の冒険者,Legendärer Abenteurer,,นักผจญภัยในตำนาน,Legendarny poszukiwacz przygód,,,Petualang Legendaris,Aventurier Légendaire,,អ្នកផ្សងព្រេង Legendary,,传奇冒险家,
 ITEM_NAME_49900002,The First Warrior,최초의 전사,O primeiro guerreiro,,まず戦士,Der erste Krieger,El primer guerrero,นักรบแรก,Pierwszy Wojownik,Pirmasis karys,De eerste Warrior,Pertama Prajurit,Le premier guerrier,Первый воин,នេះជាអ្នកចម្បាំងដំបូង,,第一勇士,
-ITEM_NAME_49900003,Lucky Adventurer,행운의 모험가,Aventureiro da sorte,,ラッキー迷惑な冒険,Glücklicher Abenteurer,Aventurero afortunado,นักผจญภัยผู้โชคดี,Lucky Adventurer.,Lucky Adventurer.,Lucky Adventurer,Lucky Adventurer.,Aventurier chanceux,Lucky Adventurer,សំណាងអ្នកផ្សងព្រេង,,幸运的冒险家,
```

Comment:
@Oremal Please don't erase the bottom three lines from this line.

Reply:
done